index.html

```diff
@@ -106,21 +129,189 @@ <h3>Terminology</h3>
+						creators are encouraged to place the rest of the publication in a subfolder (e.g., named
+							"<code>ebraille</code>"). This will make the navigation document the first HTML file users
+						encounter.</p>
+				</div>
+			</section>
+
+			<section id="fileset-naming">
+				<h3>File paths and file names</h3>
+
+				<p>eBraille file paths and file names MUST adhere to the file naming restrictions specified in <a
+						data-cite="epub-33#sec-container-filenames">section 4.2.3</a> of [[epub-33]].</p>
+			</section>
+
+			<section id="fileset-urls">
+				<h3>URLs in the file set</h3>
+
+				<p>Although the [=publication root=] establishes a common directory for all files in an [=eBraille
+					publication=], depending on how the eBraille publication is deployed it could allow references to
+					resolve outside of the publication root. For example, if an eBraille publication is deployed on the
+					web, unless it assigned its own domain, a [=path-absolute-URL string=] [[url]] (i.e., a path that
+					begins with a slash) could allow the publication to reach other resources on the server.</p>
+
+				<p>For this reason, the eBraille file set MUST NOT include file references that use path-absolute-URL
+					strings.</p>
+
+				<p>Similarly, [=eBraille reading systems=] MUST NOT resolve path-absolute-URL strings.</p>
+			</section>
+		</section>
+		<section id="ebrl-package-doc">
+			<h2>Package Document</h2>
+
+			<section id="package-doc-intro" class="informative">
+				<h3>Introduction</h3>
+
+				<p>eBraille uses the EPUB 3 <a data-cite="epub-33#sec-package-doc">package document</a> [[epub-33]] to
+					express information about an [=eBraille publication=].</p>
+
+				<p>The package document contains three primary sections:</p>
+
+				<ul>
+					<li>Metadata &#8212; the <a href="#metadata"><code>metadata</code> element</a> contains
+						bibliographic and rendering information about an eBraille publication.</li>
+					<li>Manifest &#8212; the <a href="#manifest"><code>manifest</code> element</a> contains a list of
+						all the publication resources.</li>
+					<li>Spine &#8212; the <a href="#spine"><code>spine</code></a> element contains the default ordering
+						of eBraille content documents.</li>
+				</ul>
+
+				<p>The primary difference of the eBraille package document is only in the metadata that is required and
+					recommended. The eBraille implementation of the package document is also more limited in the
+					features it allows &#8212; manifest fallbacks and legacy element are not allowed, for example.</p>
+			</section>
 
 			<section id="metadata">
 				<h3>Metadata</h3>
 
-				<div class="ednote">
-					<p>The exact method for expressing metadata in ebraille publications will be added in a future
-						update.</p>
-				</div>
+				<p>The eBraille package document metadata MUST meet all the requirements for [[epub-33]] <a
+						data-cite="epub-33#sec-pkg-metadata">metadata</a>.</p>
```

Comment:
We should also require the metadata from the EPUB Accessibility 1.1 specification.

Reply:
We haven't discussed that yet. I'm leaving that section until we get more info from the metadata folks.